src/Dialogs/Imap/ImapDialog.vala

```diff
@@ -68,6 +69,10 @@ public class OnlineAccounts.ImapDialog : Hdy.Window {
             value = 993
         };
 
+        imap_refresh_interval_spin = new Gtk.SpinButton.with_range(1, 60, 1) {
```

Comment:
```suggestion
        imap_refresh_interval_spin = new Gtk.SpinButton.with_range (1, 60, 1) {
```

We insert white space before opening parentheses or braces.

Reference
- https://docs.elementary.io/develop/writing-apps/code-style#whitespace